src/proj.cpp

```diff
@@ -228,12 +245,30 @@ Rcpp::NumericMatrix CPL_proj_direct(Rcpp::CharacterVector from_to, Rcpp::Numeric
 
 //	for (int i = 0; i < pts.nrow(); i++)
 //  		 Rcout << xx[i] << " " << yy[i] << std::endl;
-
-	if (pj_transform(fromPJ, toPJ, pts.nrow(), 0, xx.data(), yy.data(), NULL) != 0) {
-		pj_free(fromPJ); pj_free(toPJ); // #nocov start
-		Rcout << "error in pj_transform: " << pj_strerrno(*pj_get_errno_ref()) << std::endl;
-		stop("error"); // #nocov end
-	}
+        if (keep[0] == 1) {
+            // use proj_trans() on individual points, making unprojectable points be NA
+            // FIXME: not tested, since author has no access to the old proj API.
+            PJ_COORD row;
```

Comment:
remove this line?

Reply:
Oh, I'm embarrassed now. Yes, that is a remnant.